modules/carousel/jetpack-carousel.php

```diff
@@ -345,10 +345,6 @@ function enqueue_assets() {
 			wp_enqueue_style( 'jetpack-carousel', plugins_url( 'jetpack-carousel.css', __FILE__ ), array(), $this->asset_version( '20120629' ) );
 			wp_style_add_data( 'jetpack-carousel', 'rtl', 'replace' );
 
-			wp_register_style( 'jetpack-carousel-ie8fix', plugins_url( 'jetpack-carousel-ie8fix.css', __FILE__ ), array(), $this->asset_version( '20121024' ) );
```

Comment:
**Question:** Should I also remove the file from the list at [`modules/photon-cdn/jetpack-manifest.php`](https://github.com/Automattic/jetpack/blob/bd1acc8778c0e052e042b42e5574cd67317c1433/modules/photon-cdn/jetpack-manifest.php#L354)? I found it annoying to modify that file because of used line-endings and hence linter complaining about it, but happy to try.

Reply:
No need, that will be done automatically on build.

Reply:
@jeherve could the file has that info as a comment on top of it?

Reply:
That's a good idea. Adding this to my to-do list.